benchmarks/benchmark_experiment.py

```diff
@@ -58,7 +60,7 @@ def list_experiment_configs(self):
       experiment_configs.append(cfg)
     return experiment_configs
 
-  def _expand_config_choices(self, config_choices):
+  def _expand_config_choices(self, config_choices: Dict[str, List[Any]]):
```

Comment:
Shouldn't this be `List[str]`?

Reply:
Same for `_is_available` and `load_experiment`.

Reply:
I saw the config can be map to List[None] like ` "xla_flags": [None]`.  Or List[bool] like ` "keep_model_data_on_cuda": [False]`. So I keep it as `Any` Here.

Reply:
This seems like a pattern to me (also appeared in my other comment). What do you think about defining a `StrOrBool = Union[str, bool]`, and typing it as `Optional[StrOrBool]`?

Reply:
SGTM, updated all experiment_config and model_config.